File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors;

namespace DXSample {
    public partial class Main : XtraForm {
        public Main() {
            InitializeComponent();
        }

        private void OnEditValueChanged(object sender, EventArgs e) {
            treeList1.RootNodeIndent = trackBarControl1.Value;
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors

Namespace DXSample
    Partial Public Class Main
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub OnEditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles trackBarControl1.EditValueChanged
            treeList1.RootNodeIndent = trackBarControl1.Value
        End Sub
    End Class
End Namespace




